droid-hal-device.inc

```diff
@@ -425,6 +425,10 @@ sed -i -e '/^[[:space:]]*mount[[:space:]]/s/^/# Removed during droid-hal-device
 # remove the creation of /tmp to prevent wrong permissons 
 sed -i -e '/^[[:space:]]*mkdir[[:space:]]\/tmp[[:space:]]*/s/^/# Removed during droid-hal-device build : /' $RPM_BUILD_ROOT/*rc
 
+if [ $ANDROID_VERSION_MAJOR -ge "7" ]; then
+mkdir -p $RPM_BUILD_ROOT%{_libexecdir}/droid-hybris/system/etc
+cp -a %{android_root}/out/target/product/%{device}/system/etc/init $RPM_BUILD_ROOT%{_libexecdir}/droid-hybris/system/etc/.
```

Comment:
Yes we need to be careful with that script, as /system might not be yet mounted during install of dhd (especially in mic phase). Are you having some systemd .service in mind that calls a shell script on every boot that checks the state of things need to be copied/symlinked?